### ISSUE TITLE
[Feature] [Build] Non-minified Build Files

### ISSUE THREAD

**Issue by ejdaly** [hodgef/simple-keyboard#51]
Would be nice if non-minified files (simple-keyboard.js, simple-keyboard.css) were available as well as the minified versions provided in /build

Use-case: people who have their own project build steps that bundle / minify code don't necessarily want to use the already minified files; and having non-minified files are much easier to develop with / debug etc..

(Note: I tried un-minifying from index.js and index.js.map, but ended up essentially back to the development source code / modules. Would be great if the simple-keyboard.js had all the required JS in a single file, but just not minified...)

**Comment by hodgef** [comment]
Hello @ejdaly,

Thanks for providing the use-case with the request. I think it makes sense. It should be shipped sometime this week.

The end result will be as follows (for backward compatibility reasons):
```
./build
  - index.js
  - index.d.ts
  - index.js.map
  - simple-keyboard.js
  css
     - index.css
     - index.css.map
     - simple-keyboard.css
```

Regards,
Francisco Hodge

**Comment by ejdaly** [comment]
Perfect - thank you

**Comment by hodgef** [comment]
Hello @ejdaly,

The unminified dist has been released as of [2.15.0](https://github.com/hodgef/simple-keyboard/releases/tag/2.15.0)!

Feel free to take a look at the non-minified files under the [`./build`](https://github.com/hodgef/simple-keyboard/tree/master/build) folder:

- [css/simple-keyboard.css](https://github.com/hodgef/simple-keyboard/blob/master/build/css/simple-keyboard.css)
- [simple-keyboard.js](https://github.com/hodgef/simple-keyboard/blob/master/build/simple-keyboard.js)

As you can notice though, the non-minified js file is webpack's concatenated output. I adjusted the [`UglifyJS plugin`](https://github.com/mishoo/UglifyJS2) options to make it more readable, but it's not fully up to par with the source. I hope the file can still be useful in its current state.

Here's a demo where the non-minified build is linked:
https://codesandbox.io/s/github/simple-keyboard/simple-keyboard-js-demo-unminified/tree/master/

Regards,
Francisco Hodge
